src/lib/components/breadcrumbs.svelte

```diff
@@ -341,7 +344,7 @@
                                 <div use:melt={$itemProjects}>
                                     <ActionMenu.Root>
                                         <ActionMenu.Item.Anchor
-                                            href={`${base}/project-${project.region}-${project.$id}/overview/platforms`}>
+                                            href={`${base}/project-${(project.region ??= 'default')}-${project.$id}/overview/platforms`}>
```

Comment:
we shouldn't need this fallback here because `loadedProjects` should already have the fallback thats added above.